insights/specs/default.py

```diff
@@ -315,6 +315,7 @@ def semid(broker):
     lspci = simple_command("/sbin/lspci")
     lsof = simple_command("/usr/sbin/lsof")
     lssap = simple_command("/usr/sap/hostctrl/exe/lssap")
+    lsscsi = simple_command("/usr/bin/lsscsi")
```

Comment:
@JoySnow please add a `simple_file` spec to `specs/insights_archive.py` too.

Reply:
sure